packages/dds/tree/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -235,15 +235,17 @@ export class SharedTreeCore<
 	 * Update the state of the tree (including all indexes) according to the given change.
 	 * If there is not currently a transaction open, the change will be submitted to Fluid.
 	 */
-	protected applyChange(change: TChange): void {
-		const revision = mintRevisionTag();
+	protected applyChange(change: TChange, revision?: RevisionTag): void {
```

Comment:
Might be worth documenting the params here now (revision tag will be new, chosen randomly if not provided)